code_examples/core_features/did/04_full_did_simple.ts

```diff
@@ -9,6 +9,8 @@ export async function createSimpleFullDid(
   },
   signCallback: Kilt.SignExtrinsicCallback
```

Comment:
I had to set this to `Kilt.Did.GetStoreTxSignCallback` to get it running.

Reply:
Good catch! Fixed in https://github.com/KILTprotocol/docs/pull/168/commits/dddb0da1fe62226653c97b07219dfcbc10bc5af2.